src/index.ts

```diff
@@ -1,82 +1,49 @@
-import express from 'express';
+import express, { Application } from 'express';
 import router from './routes';
 import busboy from 'connect-busboy';
 import { PUBLIC_FOLDER, PORT } from './config';
-import { serviceRunner } from 'services';
-import { HeartBeatService } from 'services/heartBeat';
-import { InitCronService } from 'services/initCron';
-import { GnssHealthCheck } from 'services/gnssHealthCheck';
-import { UpdateCameraConfigService } from 'services/updateCameraConfig';
-import { DeviceInfoService } from 'services/deviceInfo';
-import { TrackDownloadDebt } from 'services/trackDownloadDebt';
-import { setSessionId } from 'util/index';
 import { initUbxSessionAndSignatures } from 'ubx/session';
 import console_stamp from 'console-stamp';
+import { runServices } from 'services/runner';
 
 //import { BootNetworkService } from 'services/bootNetwork';
 // import { AssistNowService } from 'services/assistNow';
 
-export async function initAppServer() {
-  const app = express();
+const app: Application = express();
 
-  // Making all the files accessible via direct HTTP urls
-  app.use('/public', express.static(PUBLIC_FOLDER));
-  app.use(
-    busboy({
-      highWaterMark: 2 * 1024 * 1024, // Set 2MiB buffer
-    }),
-  ); // Handles file uploads for Over-The-Air update
-  app.use(express.json());
-  app.use(express.urlencoded({ extended: true }));
-
-  app.use(router);
+// Making all the files accessible via direct HTTP urls
+app.use('/public', express.static(PUBLIC_FOLDER));
+app.use(
+  busboy({
+    highWaterMark: 2 * 1024 * 1024, // Set 2MiB buffer
+  }),
+); // Handles file uploads for Over-The-Air update
+app.use(express.json());
+app.use(express.urlencoded({ extended: true }));
+app.use(router);
 
+(async () => {
   await new Promise<void>((resolve, reject) => {
```

Comment:
@punov This line looks suspicious. Is there a need to wrap this synchronous operation with a Promise that will resolve right away? 

I think the only effect would be to send this operation to the event loop (micro). In other words, wait until current stack execution is finished, then execute this line. Should it just go at the end then?

Reply:
Is this change related to the new endpoint at all @h-tohkubo?